cassandra/io/libevwrapper.c

```diff
@@ -667,9 +667,11 @@ initlibevwrapper(void)
     if (PyModule_AddObject(module, "Timer", (PyObject *)&libevwrapper_TimerType) == -1)
         INITERROR;
 
+    #if PY_VERSION_HEX < 0x030700F0
     if (!PyEval_ThreadsInitialized()) {
         PyEval_InitThreads();
     }
+    #endif
```

Comment:
What version does this guard check against? Is it 3.6? If its 3.6 why not just remove the call instead of guarding it? We no longer support 3.6.

Reply:
I would just compare with cython generated code, and copy the logic from there

Reply:
True, removed.